configure.src

```diff
@@ -148,7 +148,14 @@ while test $# != 0; do
 	    case $1 in
 		--without-*)
 		    skip_app=`expr "$1" : '--without-\(.*\)'`
-		    if test -d "lib/$skip_app"; then
+                    if [ "$skip_app" = "stdlib" ] ||
+                           [ "$skip_app" = "kernel" ] ||
+                           [ "$skip_app" = "sasl" ] ||
+                           [ "$skip_app" = "compiler" ] ||
+                           [ "$skip_app" = "erls_interface" ]; then
```

Comment:
I believe it's supposed to be "erl_interface".

Reply:
Just wanted to check if anyone reads my code ;) will fix! Thanks!